cron/docker-compose.yml

```
@@ -0,0 +1,17 @@
+version: '3'
+
+services:
+  movies-cron:
+    build: .
+    container_name: movies-cron
+    restart: unless-stopped
+    volumes:
+      - /data:/data
```

la justement je pense que faudrait tenter/tester de mettre :
```suggestion
      - ../data/:/data/
```